patches/libssp/0001-allow-building-outside-of-the-gcc-tree.patch

```diff
@@ -0,0 +1,124 @@
+From 08fc23c3581c13ff0d749a2565d134b337995bff Mon Sep 17 00:00:00 2001
+From: Steve Lhomme <robux4@ycbcr.xyz>
+Date: Tue, 7 Apr 2020 13:56:58 +0200
+Subject: [PATCH 1/2] allow building outside of the gcc tree
```

Comment:
It looks like this commit is missing the changes to build-libssh.sh.

Also, I'd suggest changing the commit message to "build-libssp: Build using the upstream autotools build system"

Reply:
Fixed